ckanext/validation/commands.py

```diff
@@ -162,8 +161,8 @@ def run_validation(self):
 
             elif not self.options.assume_yes:
 
-                msg = ('\nYou are about to start validation for {0} datasets' +
-                       '.\n Do you want to continue?')
+                msg = ('\nYou are about to start validation for {0} datasets'
+                       + '.\n Do you want to continue?')
```

Comment:
The `+` isn't actually needed, Python concatenates them automatically.

It would also make more sense to have the full stop on the previous line.